wp-content/plugins/wporg-learn/inc/export.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * Allow some raw data to be exposed in the REST API for certain post types, so that developers can import
+ * a copy of production data for local testing.
+ */
+
+namespace WPOrg_Learn\Export;
+
+defined( 'WPINC' ) || die();
+
+add_action( 'rest_api_init', function() {
+	// Important: only expose raw post content for specific post types
+	register_raw_content_for_post_type( 'lesson-plan' );
+	register_raw_content_for_post_type( 'wporg_workshop' );
+} );
+
+
+
+function register_raw_content_for_post_type( $post_type ) {
+
+	register_rest_field(
+		$post_type,
+		'content_raw',
+		[
+			'get_callback' => __NAMESPACE__.'\show_post_content_raw',
+			'schema'       => [
+				'type' => 'string',
+				'context' => [ 'export' ]
+			]
+		]
+	);
+
+	add_filter( "rest_{$post_type}_item_schema", __NAMESPACE__.'\add_export_context_to_schema' );
+}
+
+// Filter a CPT item schema and make it so that every item with 'view' context also has 'export' context.
+function add_export_context_to_schema( $schema ) {
+	update_schema_array_recursive( $schema );
+
+	return $schema;
+}
+
+// Find every item in the schema that has a 'view' context, and add an 'export' context to it.
+// Had to use a recursive function because array_walk_recursive only walks leaf nodes.
+function update_schema_array_recursive( &$schema ) {
+	foreach ( $schema as $key => &$value ) {
+		// Head recursion
+		if ( is_array( $value ) ) {
+			update_schema_array_recursive( $value );
+		}
+		if ( 'context' === $key && in_array( 'view', $value ) ) {
+			$value[] = 'export';
+		}
+	}
+}
+
+function get_all_block_names( $blocks ) {
+	$block_names = [];
+
```

Comment:
```suggestion
    if ( ! $blocks ) {
        return array();
    }
```

I got an error on my sandbox w/ out this, because `$blocks` was `null` for https://learn.wordpress.org/wp-json/wp/v2/wporg_workshop?context=export

The workshop content is in the response, though, so I'm guessing it's just empty in one of the recursive calls on some posts, or something like that.